jekyll/_cci2/webhooks.md

```diff
@@ -0,0 +1,198 @@
+| id    | yes             | Unique ID of the organization              |
+| name  | yes             | Name of the organization (e.g. "circleci") |
+{: class="table table-striped"}
+    
+### Job
+{: #job}
+
+A job typically represents one phase in a CircleCI workload (e.g. "build", "test", or "deploy") and contains a series of steps.
+
+| Field  | Always present? | Description                                                                                                  |
+|--------|-----------------|--------------------------------------------------------------------------------------------------------------|
+| id     | yes             | Unique ID of the job                                                                                         |
+| number | yes             | An auto-incrementing number for the job, sometimes used in CircleCI's APIs to identify jobs within a project |
+| name   | yes             | Name of the job as defined in .circleci/config.yml                                                           |
+| status | yes             | Current status of the job                                                                                    |
+{: class="table table-striped"}
+
+
+### Workflow
+{: #workflow}
+
+Workflows contain many jobs, which can run in parallel and/or have dependencies
+between them. A single git-push can trigger zero or more workflows, depending on
+the CircleCI configuration (but typically one will be triggered).
+
+
+| Field       | Always present? | Description                                                        |
+|-------------|-----------------|--------------------------------------------------------------------|
+| id          | Yes             | Unique ID of the workflow                                          |
+| name        | Yes             | Name of the workflow as defined in .circleci/config.yml            |
+| status      | No              | Current status of the workflow. Not included in job-level webhooks |
+| created\_at | Yes             | When the workflow was created                                      |
+| stopped_at  | No              | When the workflow reached a terminal state (if applicable)         |
+| url         | Yes             | URL to the workflow in CircleCI's UI                               |
+{: class="table table-striped"}
+
+### Pipeline
+{: #pipeline}
+
+Pipelines are the most high-level unit of work, and contain zero or
+more workflows. A single git-push always triggers up to one pipeline. Pipelines
+can also be triggered manually through the API.
+
+A pipeline's overall status doesn't change after it is created, so there are no
+exclusively-pipeline-related events.
+
+Re-running a workflow (either via "Re-run from failed" or "Re-run from
+beginning") creates a new, distinct workflow in the same pipeline.
+
+A pipeline may have parameters supplied during a manual invocation, that affect
+the behavior of its workflows and jobs.
```

Comment:
When I wrote the introductory paragraphs for these three sections (Job, Workflow, Pipeline), it was mostly for the benefit of our initial partners who needed to know how our entity model compares to other CI providers. That context may not be necessary for the typical reader of this page. What do you think about keeping these descriptions short and linking to other relevant docs within the site?

Reply:
I'm a fan of Alex's idea if you think it makes sense @teesloane.

Reply:
Yeah, I can definitely trim this down - I think I would like to leave a brief description of pipelines (as we don't have a page dedicated to what a pipeline is exactly 😓 ) and then we'll update the page when there is one with a proper link.